packages/dataviews/CHANGELOG.automattic.md

```diff
@@ -4,6 +4,7 @@
 
 ## 0.2.0
 
+- `text`, `email` Edit control: add `help` support from the field `description` prop.
```

Comment:
It should be Next section 🤔

Reply:
Ah yes, true. https://github.com/Automattic/wp-calypso/pull/103726/files